src/control/server/server.go

```diff
@@ -225,6 +225,29 @@ func Start(log *logging.LeveledLogger, cfg *config.Server) error {
 		control.WithConfig(cliCfg),
 		control.WithClientLogger(log))
 
+	// Init management RPC subsystem.
+	mgmtSvc := newMgmtSvc(harness, membership, sysdb)
+	eventPubSub := events.NewPubSub(ctx, log)
+	defer eventPubSub.Close()
+
+	// Forward received events to management service by default.
+	eventForwarder := control.NewEventForwarder(rpcClient, cfg.AccessPoints)
+	eventPubSub.Subscribe(events.RASTypeRankStateChange, eventForwarder)
+
+	mgmtSvc.dispatchEvents = func(events ...events.Event) {
+		for _, event := range events {
+			eventPubSub.Publish(event)
+		}
+	}
+	mgmtSvc.disableEvents = func(ids ...events.RASID) {
+		eventPubSub.AddToMask(ids...)
+	}
+	mgmtSvc.enableEvents = func(ids ...events.RASID) {
+		eventPubSub.RemoveFromMask(ids...)
+	}
+
```

Comment:
IMO these should be set in some constructor, not on start

Reply:
moved the eventPubSub into the mgmtSvc and enable events function called directly now